tequilapi/contract/tokens.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2022 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package contract
+
+import (
+	"math/big"
+
+	"github.com/mysteriumnetwork/payments/crypto"
+)
+
+// TokensZero zero amount
+var TokensZero = Tokens{Wei: "0", Ether: "0", Human: "0"}
+
+// HumanPrecision default human ether amount precision
+const HumanPrecision = 6
+
+// Tokens a common response for ethereum blockchain monetary amount
+type Tokens struct {
+	Wei   string `json:"wei"`
+	Ether string `json:"ether"`
+	Human string `json:"human"`
+}
+
+// NewTokens convenience constructor for Tokens
+func NewTokens(amount *big.Int) Tokens {
+	if amount == nil {
+		return TokensZero
+	}
+
+	ethers := crypto.BigMystToDecimal(amount)
+
+	return Tokens{
+		Wei:   amount.String(),
+		Ether: ethers.String(),
+		Human: ethers.Truncate(HumanPrecision).String(),
```

Comment:
Imo the human value here is not something backend should be concerned about. It's not like API should ever consume it in order to specify some amount of tokens and if we want to display it nicely, apps should just convert it to some notation they think is correct. 🤷🏻 

When it comes to any interaction with the API we should only really accept `Wei`.

Reply:
@tadaskay your position?

Reply:
We already have this in android API and it has helped a lot to avoid cumbersome logic on client side. Because API is an interface, the most important thing is how easy it is to use it from the consumer point of view. I shouldn't need to deal with the monstrosity where 1 token is presented as a number with 18 zeroes. So thanks for opinion, but this PR solves this long lasting issue.